Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/SessionManager.cs

```diff
@@ -179,7 +186,7 @@ public ConnectStatus OnClientApprovalCheck(ulong clientId, string clientGUID, T
                 return GetPlayerData(clientGUID);
             }
 
-            Debug.Log("No client guid found mapped to the given client ID");
+            Debug.LogError("No client guid found mapped to the given client ID");
```

Comment:
For this and the other two error logs below, I'd also log the client Id passed in.